src/test/java/dog/pawbook/testutil/TypicalIndexes.java

```diff
@@ -7,7 +7,10 @@
     public static final Integer ID_FIRST_OWNER = 1;
     public static final Integer ID_SECOND_OWNER = 2;
     public static final Integer ID_THIRD_OWNER = 3;
-    public static final Integer ID_FIRST_DOG = 1;
-    public static final Integer ID_SECOND_DOG = 2;
-    public static final Integer ID_THIRD_DOG = 3;
+    public static final Integer ID_FIRST_DOG = 2;
+    public static final Integer ID_SECOND_DOG = 3;
+    public static final Integer ID_THIRD_DOG = 4;
+    public static final Integer ID_FIRST_PROGRAM = 1;
+    public static final Integer ID_SECOND_PROGRAM = 2;
+    public static final Integer ID_THIRD_PROGRAM = 3;
```

Comment:
originally this file worked together with only one typical addressbook, but now there is 3 copies on top of the fact that there is no longer ID, I think it will be better to define these constants directly within each addressbook to avoid confusion. Each of those addressbooks require better naming as well as they are not descriptive at all and there is nothing typical about them. To make it more typical, perhaps it is good to just combine 3 of those into a `TypicalEntities` class instead. Something like `getOwnersOnlyAddressBook` and `getSingleOwnerMultipleDogsAddressBook` will help make things clear.